### PR TITLE
[MERGE] [FE] Merge CreateRestoQR -> Main

### PR DESCRIPTION
**Pull request by Alban-de-tourtier**
### 🛠 Changes being made

I added QRcode call on the BE and I added a button on the restoWeb and modify visitor menu URL

### ✨ What's the context?

PLD

### 🧪 Test plan

All good

### 🏎 PR checklist

1. Read the ticket
- [x] Read the related ticket on Jira and understand the requirements for this ticket. In case anything is unclear, ask the ticket creator or ticket executor.
2. Matching PR description
- [x] Check that the PR description on GitHub is aligned with the ticket description on Jira.
3. Review the changes on GitHub
- [x] is the current version of the destination branch (mostly main) merged into the PR branch?
- [x] does it fulfill the definition of done for the ticket?
- [x] were the tests adjusted or extended for the changed behavior?
- [x] was the documentation adjusted or extended for changed behavior?
- [x] does it align with the team code guidelines?
- [x] check readability of code - is it easy to understand what is happening in the code snippets?
- [x] proper naming
- [x] is it logical?
- [x] are there any performance issues?
- [x] could it be improved / simplified?
4. Execute locally / in testing environment
- [x] Execute the code locally and check if there are any issues. Test whether the new behavior from the ticket acts as expected. Try to think of edge cases and test them.
5. Run tests
- [x] Run the tests locally and check that all of them succeed.
6. Linting & Formatting
- [x] Check that the changes didn’t cause any linting or formatting warnings of errors.
7. Submit Feedback
- [x] Approve if all of the following points fulfilled:
   - all previous steps did not reveal any issues
   - there are no open comments on the PR

- [x] otherwise, request changes with specific comments (constructive) on what to change!

### CODE REVIEW

**Review by RenanDbs — APPROVED** [review]

File: packages/restoWeb/src/pages/MenuPage/MenuPage.tsx [hunk 1]
```diff
@@ -59,6 +67,14 @@ const MenuPage = () => {
             )
           );
         })}
+        <Button
+          className={styles.SaveBtn}
+          variant="contained"
+          sx={{width: "12.13rem"}}
+          onClick={() => window.location.href = `${process.env.DB_HOST}${process.env.DB_HOST_PORT}/api/qrcode/base64/${URL.name}`}
+        >
+          Get my Menu QRCODE
```
Comment (JosefineM2000) [inline packages/restoWeb/src/pages/MenuPage/MenuPage.tsx:76]: Please use i18n here

**Review by JosefineM2000 — CHANGES_REQUESTED** [review]

**Review by JosefineM2000 — APPROVED** [review]
